### DISCUSSION TITLE
Antibiotic resistant genes (Multi-drug resistant IDs)

### DISCUSSION THREAD

**Discussion by ramin-karimian** [geneontology/helpdesk#386] [Q&A]
How to identify Antibiotic resistant genes (Multi-drug resistant IDs) from GO database?

**Comment by suzialeksander** [comment]
Hi @ramin-karimian, I think we responded to this on Biostars and gave you a few options there.  Did you have any further questions about GO? I think one of the other users mentioned CARD, which I am not familiar with but may be a good resource for your work.

**Comment by ramin-karimian** [comment]
Hi,

I checked the response for my question on Biostars. It makes sense. Many
thanks for your reply.

Best regards,
Ramin
